src/jsifier.js

```diff
@@ -260,8 +279,14 @@ function ${name}(${args}) {
 
       if (symbolsOnly) {
         if (!isJsOnlySymbol(symbol) && LibraryManager.library.hasOwnProperty(symbol)) {
-          externalDeps = deps.filter((d) => !isJsOnlySymbol(d) && !(d in LibraryManager.library) && typeof d === 'string');
-          symbolDeps[symbol] = externalDeps;
+          var value = LibraryManager.library[symbol];
+          var resolvedSymbol = symbol;
+          // Resolve aliases before looking up deps
+          if (typeof value == 'string' && value[0] != '=' && LibraryManager.library.hasOwnProperty(value)) {
+            resolvedSymbol = value;
+          }
+          var transtiveDeps = getTransitiveDeps(resolvedSymbol);
```

Comment:
This can take quadratic time, I think? If we have a chain `a->b->c->..` and we compute first `a`, which computes all the children, then `b`, etc., we'd be doing a lot of extra work. Doing a single fixpoint flow up front might be faster. Then again, maybe this is fast enough already?

Reply:
I'm not sure it worth the complexity at this point.  I measured the cost of JS symbol generation (running `compiler.js --symbol-only`) both with this code and with `getTransitiveDeps` simply removed.

Without `getTransitiveDeps` the average of 3 runs was 0.236s.  With it was 0.225s.   So this is taking just under 1% of the JS symbol generation time.   (I think the reason is that dependencies are not particularly common within the JS library.. I would imagine the average number of deps is under 1.. plus the js compiler is doing much more expensive stuff than this).

Reply:
Maybe the function `getTransitiveDeps` could memoize its computation inside the loop?

It would be great to have e.g. emprofile runs print out the time block this takes, though since this is in js code and not in top level python code, I suppose there is no good way to specifically block out this subprocedure into those diagrams?

Reply:
Yes, some kind of Dynamic Programming / Memoization would be the a logical way to speed this up.   At this point that seems a little premature though since its taking less than 1% of the JS compile time.. and we already cache the results of the JS compile.